docs/admin/reference/responsibilities.rst

```diff
@@ -0,0 +1,224 @@
+Responsibilities
+================
+
+.. _Responsibilities:
+
+The SecureDrop architecture contains multiple machines and hardened servers.
+While many of the installation and maintenance tasks have been automated, a
+skilled Linux admin is required to responsibly run the system.
+
+Responsibilities of SecureDrop administrators
+---------------------------------------------
+As a SecureDrop administrator, it is your responsibility to:
+
+* :ref:`manage users <manage_users>`
```

Comment:
In this new structure, I would suggest we add a bullet point "install SecureDrop" linking to the installation guide.